### DISCUSSION TITLE
Episode 128: GCC 13.1 🥳 and Upcoming Conferences!

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#19] [Episode]
Episode Link: https://adspthepodcast.com/2023/05/05/Episode-128.html

Feel free to write comments or ask questions 🙂

**Comment by joblobob** [comment]
I'll see you at CppNorth!! I'm also excited about the new C++23 algos!

**Reply by codereport** [reply]
Looking forward to seeing you there!
